File: app/src/main/java/com/gee12/mytetroid/data/IRecordFileCrypter.java
package com.gee12.mytetroid.data;

import android.content.Context;

import com.gee12.mytetroid.model.TetroidRecord;

public interface IRecordFileCrypter {
    boolean cryptRecordFiles(Context context, TetroidRecord record, boolean isCrypted, boolean isEncrypt);
}

File: app/src/main/java/com/gee12/mytetroid/data/crypt/IRecordFileCrypter.kt
package com.gee12.mytetroid.data.crypt

import android.content.Context
import com.gee12.mytetroid.model.TetroidRecord

interface IRecordFileCrypter {
    suspend fun cryptRecordFiles(context: Context, record: TetroidRecord, isCrypted: Boolean, isEncrypt: Boolean): Boolean
}